backend/src/lib/json-request.ts

```diff
@@ -50,3 +58,39 @@ export function jsonPost<T = unknown>(
     return result
   })
 }
+
+export function jsonPut(url: string, body: unknown, token?: string): Promise<PutResponse> {
+  const headers: HeadersInit = {
+    [HTTP2_HEADER_CONTENT_TYPE]: 'application/json',
+  }
+  if (token) headers[HTTP2_HEADER_AUTHORIZATION] = `Bearer ${token}`
+  return fetchRetry(url, {
+    method: 'PUT',
+    headers,
+    agent: new Agent({ rejectUnauthorized: false }),
```

Comment:
We should not be ignoring SSL errors by default. If this is needed, it needs to be captured in the threat model and documented as a security risk.

Reply:
I updated the agent to `agent: new Agent({ ca: getCACertificate() })` so the rejectUnauthorized will default to true, and the ca-cert will be overridden.